app/ide-desktop/lib/dashboard/src/components/AriaComponents/Dialog/utilities.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file
+ *
+ * This file contains a function that checks if the element is a part of a component that should ignore the interact outside event.
+ */
+
+import * as React from 'react'
+
+import * as eventCallback from '#/hooks/eventCallbackHooks'
+
+import * as aria from '#/components/aria'
+
+import * as dialogStackProvider from './DialogStackProvider'
+
+const IGNORE_INTERACT_OUTSIDE_ELEMENTS = [
+  // Toastify toasts
+  '.Toastify__toast-container',
+  // ReactQuery devtools
+  '.tsqd-parent-container',
+  '[data-enso-ignore-click-outside]',
+]
+
+/**
+ * Check if the element is a part of a component that should ignore the interact outside event
+ */
+export function shouldIgnoreInteractOutside(element: HTMLElement) {
+  return IGNORE_INTERACT_OUTSIDE_ELEMENTS.some(selector => element.closest(selector))
```

Comment:
is there a reason we don't use just a single selector using `:is()`?

Reply:
No reason, will try!